src/components/SlateEditor/PlainTextEditor.jsx

```diff
@@ -10,13 +10,30 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Editor } from 'slate-react';
 import Types from 'slate-prop-types';
+import isHotkey from 'is-hotkey';
+
+const isSaveHotkey = isHotkey('mod+s');
```

Comment:
Har det noko å sei at du bruker annen kode enn i RichTextEditor der det brukes `import { isKeyHotkey } from 'is-hotkey';`

Reply:
Jeg vet ikke helt hva forskjellen på isHotkey og isKeyHotkey er, men jeg kan skrive det om så vi bruker isKeyHotkey begge steder.

Reply:
Det er også ett gjenstående problem under `topicArticle` hvor innholdsboksen oppfører seg slik ingressfeltet gjorde og låser seg. Dette må fikses før denne kan merges. (Se: [https://editorial-frontend-pr-884.ndla.sh/subject-matter/topic-article/new](https://editorial-frontend-pr-884.ndla.sh/subject-matter/topic-article/new
Dersom man prøver å trykke i _innhold_ så får man ikke flyttet fokus)

Reply:
Funka når eg brukte samme parameterene som for læringsressurs.